### PR TITLE
[zh] Sync Ambient K3S prerequisite #14826 into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync #14826 into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

**Comment by wilsonwu** [comment]
The k3s Chinese doc anchor is not working, submit PR to fix: https://github.com/k3s-io/docs/pull/261

File: content/zh/docs/ops/ambient/install/platform-prerequisites/index.md [hunk 1]
```diff
@@ -24,6 +24,14 @@ test: no
    则必须在 `helm install` 命令附加
    `--set values.cni.cniConfDir=/var/snap/microk8s/current/args/cni-network --set values.cni.cniBinDir=/var/snap/microk8s/current/opt/cni/bin`。
 
+### K3S {#k3s}
+
+1. 如果您使用的是 [K3S](https://k3s.io/)，
+   则必须在 `helm install` 命令中附加
+   `--set values.cni.cniConfDir=/var/lib/rancher/k3s/agent/etc/cni/net.d --set values.cni.cniBinDir=/var/lib/rancher/k3s/data/current/bin/`，
+   因为 K3S 使用非标准位置来存储 CNI 配置和二进制文件。
+   这些非标准位置也可能[根据 K3S 文档](https://docs.k3s.io/zh/cli/server#k3s-server-cli-help)被覆盖。
```
Comment (wilsonwu) [inline content/zh/docs/ops/ambient/install/platform-prerequisites/index.md:33]:
```suggestion
   这些非标准位置也可能[根据 K3S 文档](https://docs.k3s.io/zh/cli/server#k3s-server-cli-%E5%B8%AE%E5%8A%A9)被覆盖。
```

File: content/zh/docs/ops/ambient/install/platform-prerequisites/index.md [hunk 2]
```diff
@@ -24,6 +24,14 @@ test: no
    则必须在 `helm install` 命令附加
    `--set values.cni.cniConfDir=/var/snap/microk8s/current/args/cni-network --set values.cni.cniBinDir=/var/snap/microk8s/current/opt/cni/bin`。
 
+### K3S {#k3s}
+
+1. 如果您使用的是 [K3S](https://k3s.io/)，
+   则必须在 `helm install` 命令中附加
+   `--set values.cni.cniConfDir=/var/lib/rancher/k3s/agent/etc/cni/net.d --set values.cni.cniBinDir=/var/lib/rancher/k3s/data/current/bin/`，
+   因为 K3S 使用非标准位置来存储 CNI 配置和二进制文件。
+   这些非标准位置也可能[根据 K3S 文档](https://docs.k3s.io/zh/cli/server#k3s-server-cli-%E5%B8%AE%E5%8A%A9)被覆盖。
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/install/platform-prerequisites/index.md:33]:
```suggestion
   [根据 K3S 文档](https://docs.k3s.io/zh/cli/server#k3s-server-cli-%E5%B8%AE%E5%8A%A9)这些非标准位置也可以被覆盖。
```

**Review by windsonsea — COMMENTED** [review]
override 是覆盖，还是重载？

**Comment by wilsonwu** [comment]
> override 是覆盖，还是重载？

应该是覆盖或重写的意思。

**Review by windsonsea — APPROVED** [review]
Thanks

**Pull request merged**